renovate.json

```diff
@@ -0,0 +1,73 @@
+{
+  "extends": [
+    "config:js-lib",
+    ":pinOnlyDevDependencies",
+    ":automergePatch",
+    ":automergeMinor",
+    ":semanticCommits"
+  ],
+  "timezone": "America/Los_Angeles",
+  "schedule": ["after 3am", "before 9am"],
```

Comment:
This is useful to avoid potentially rate limiting your github user during business hours (I have this problem w/ renovate set up on ~60 libs)